src/services/backupInstances/format.ts

```diff
@@ -0,0 +1,107 @@
+import cuid from 'cuid'
+import { isEmpty } from 'lodash'
+import {
+  AzureBackupInstance,
+  AzureBackupInstanceProperties
+} from '../../types/generated'
+import { transformSystemData } from '../../utils/format'
+import { RawAzureBackupInstanceResource } from './data'
+import { BackupInstance } from './utils'
+
+const formatProperties = (
+  properties?: BackupInstance
+): AzureBackupInstanceProperties => {
+  if (isEmpty(properties)) {
+    return {}
+  }
+  const {
+    policyInfo = {},
+    protectionErrorDetails = {},
+    protectionStatus = {},
+    ...rest
+  } = properties
+
+  return {
+    policyInfo: {
+      policyId: policyInfo?.policyId,
+      policyParameters: {
+        dataStoreParametersList:
+          policyInfo?.policyParameters?.dataStoreParametersList?.map(p => ({
+            id: cuid(),
+            ...p,
```

Comment:
avoid spreading properties, if the sdk returns values that arent in the type def (which it has before) or there are new properties added, the insert will fail because the schema is wrong.